### PR TITLE
FC-0068 docs: diataxis criteria documentation

### PR DESCRIPTION
**Pull request by Apgomeznext**
This PR adds the diataxis criteria documentation approved.

Google doc: https://docs.google.com/document/d/1mHnd2mJTy-IAgGxE0sP6AO0_oNogQbXhEQNZtzigLww/edit?tab=t.0#heading=h.lpo0vmwalr3v

Rendered page: https://docsopenedxorg--612.org.readthedocs.build/en/612/documentors/concepts/content_types.html

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Apgomeznext!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.


#### :radio_button: Update the status of your PR

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.




#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:11]: Recommend linking to Diataxis homepage (I know you do it in a Note at the end, but that's pretty far away from here):

```suggestion
The `Diataxis framework <https://diataxis.fr/>`_ is an approach to quality in technical documentation and creates a systematic organization. Diataxis identifies four modes of documentation:
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:18]:
```suggestion
Using Diataxis, documentation can be organized to meet diverse user needs, making it easier to find relevant information based on the user’s situation or goal. This structure enhances usability and comprehension, allowing users to navigate documentation more effectively.
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:20]:
```suggestion
For each type of document, you will find the explanation, the technical guidelines, and a matrix of criteria that will allow you to identify whether or not a document complies with the parameters of the Diataxis used in the Open edX repositories. 
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:27]:
```suggestion
A Quickstart is a lesson that guides the reader through **practical steps** to complete a stated goal. It is **learning-oriented**. In other words, it is a lesson and a learning experience. For example, creating a virtual course can be a Quickstart because it’s a practical skill. A Quickstart may comprise the contents of multiple How-To guides.
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:52]: This is used as an example below of a How-To topic. Maybe

```suggestion
	E.g., *“You have successfully created your first course.”*
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:130]: The numbering is off here:

image

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:142]:
```suggestion
By following these recommendations, you will be able to write a good quality how-to guide:
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:226]: I puzzled over this sentence a bit. I think this is more accurate but please let me know if you meant something else.

```suggestion
- **The documentation structure should mirror the product's structure** so the user can work their way through it simultaneously. It doesn’t mean forcing the documentation into an unnatural structure. What’s important is that the logical, conceptual arrangement of and relations within the documentation should help make sense of the product.
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:286]: Numbering is off

image

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:296]:
```suggestion
- **Concept guides are about a topic.** You should be able to place an implicit (or even explicit) "About" before each title—for example, “(About) Instructional Design.” Also, concept document names should use nouns or noun phrases that indicate theoretical or conceptual topics.
```

File: source/documentors/concepts/content_types.rst
Comment (sarina) [inline source/documentors/concepts/content_types.rst:341]:
```suggestion
.. Note:: To learn more about the Diataxis framework, visit `the Diataxis homepage <https://diataxis.fr/>`_. The Open edX community adapted the framework according to the platform's needs.
```

**Review by sarina — CHANGES_REQUESTED** [review]

**Comment by sarina** [comment]
I edited your pull request description with a link to the Google doc as well as to the rendered page, which is helpful for both reviewers and anyone who comes across this PR at a later time.

**Review by jesgreco — APPROVED** [review]

**Pull request merged**
